MAMchem_GridComp/optics/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+set(miev_srcs 
+   miev/ErrPack.f
+   miev/MIEV0.F
+   )
+
+add_library(miev ${miev_srcs})
+target_link_libraries(miev PUBLIC OpenMP::OpenMP_Fortran)
```

Comment:
Didn't we make dependency on OpenMP conditional to support NOAA?

Reply:
Is NOAA using our full Chem GridComp? Or just GOCART?

Reply:
Fair point, but ... I thought we were generally trying to avoid the dependency and were opening it up for NOAA (under a conditional).  Certainly no harm here.    I should formulate a genera rule I suppose, but ... its a mess.